src/containers/Review/ReviewSubmit.tsx

```diff
@@ -161,7 +161,9 @@ const ReviewSubmitButton: React.FC<ReviewSubmitProps & ReviewSubmitButtonProps>
   }
 
   const onClick = async () => {
-    const firstIncompleteReviewPage = structure.firstIncompleteReviewPage
+    const firstIncompleteReviewPage = assignment.isFinalDecision
+      ? null
+      : structure.firstIncompleteReviewPage
```

Comment:
Hmmm seems like a slip from different PR?

Reply:
Okay, it should be all good now.